src/ai/pull-request-workflow.ts

```diff
@@ -231,6 +231,23 @@ async function revisePullRequest(
     );
     throw error;
   }
+
+  // Check if we need to revert the last commit
+  if (await needsRevert(prDetails)) {
+    console.log(chalk.yellow('Reverting last commit as requested...'));
+    await revertLastCommit(basePath);
+    await githubAPI.pushChanges(owner, repo, prDetails.head.ref);
+    await githubAPI.addCommentToPR(
```

Comment:
addCommentToPR doesn't really work here. implement a new function that takes any string to post as a comment and use it here.